rclcpp/include/rclcpp/node.hpp

```diff
@@ -663,6 +663,16 @@ class Node : public std::enable_shared_from_this<Node>
   const NodeOptions &
   get_node_options() const;
 
+  /// Manually assert that this Node is alive (for RMW_QOS_POLICY_LIVELINESS_MANUAL_BY_NODE)
+  /**
+   * If the rmw Liveliness policy is set to RMW_QOS_POLICY_LIVELINESS_MANUAL_BY_NODE, the creator
+   * of this node may manually call `assert_liveliness` at some point in time to signal to the rest
```

Comment:
What do you think about replacing `manually` with `periodically`?

Reply:
And if so, drop `at some point in time`.

Reply:
Making manual calls to `assert_liveliness()` is not necessarily periodic. Some node activities implies liveliness, "Consequently the use of assert_liveliness is only needed if the application is not writing data regularly" (from the spec).

Reply:
Oh ok, I didn't think about the piggy-backing on data. Sounds good to me, thanks for explaining it.